cnudie/upload.py

```diff
@@ -110,10 +110,7 @@ def upload_component_descriptor(
     component = component_descriptor.component
 
     schema_version = component_descriptor.meta.schemaVersion
-    if isinstance(schema_version, enum.Enum):
-        # hack: convert from gci.componentmodel
-        schema_version = ocm.SchemaVersion(schema_version.value)
-    else:
+    if not isinstance(schema_version, ocm.SchemaVersion):
```

Comment:
nit: You can drop this check and always try to parse to `ocm.SchemaVersion` (it doesn't make a difference if the object is already of this type).